src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -487,7 +487,25 @@ NodeResources ToNodeResources(const NodeResourceInstances &instance) {
 
 }  // namespace
 
+void LocalResourceManager::UpdateAvailableObjectStoreMemResource() {
+  // Update local object store usage and report to other raylets.
+  if (get_used_object_store_memory_ == nullptr) {
+    return;
+  }
+
+  auto &capacity = local_resources_.predefined_resources[OBJECT_STORE_MEM];
+  RAY_CHECK_EQ(capacity.total.size(), 1);
+
+  const double used = get_used_object_store_memory_();
+  const double total = capacity.total[0].Double();
+  capacity.available[0] = FixedPoint(total >= used ? total - used : 0.0);
+
+  OnResourceChanged();
```

Comment:
ah... did i miss this during refactor?

Reply:
Oh, no. This PR was reverted because at that time, I didn't have the locality PR. Since now locality PR is merged, I want to retry it.